s3_bucket.go

```diff
@@ -1323,3 +1325,64 @@ func (s *s3ArchiveBucket) Pull(ctx context.Context, opts SyncOptions) error {
 
 	return nil
 }
+
+// PresignExpireTime sets the amount of time the link is live before expiring.
+const PresignExpireTime = 24 * time.Hour
+
+// RequestParams holds all the parameters needed to sign a url or fetch headObject.
+type RequestParams struct {
```

Comment:
Can we give this a more descriptive name? This package has a lot of implementations of the `Bucket` interface and  in consequence a lot of options structs.

Reply:
I changed it to PreSignRequestParams. Does that work?

Reply:
Thanks!